specification/common-types/resource-management/v1/types.json

```diff
@@ -120,6 +120,13 @@
           ],
           "description": "The geo-location where the resource lives"
         },
+        "extendedLocation": {
```

Comment:
This changes every single resource from every RP in every api-version that references ResourceModelWithAllowedPropertySet - in other words, it is potentially a massive breaking change across all RPs.  We cannot simply add a new property to the resource definitions in common types.

We should likely add a new resource type, or create a new version of common-types.

Also, this modelling is strange - you are creating a new inline schema definition that extends ExtendedLocation, rather than just referencing the ExtendedLocation schema.  When you do model this in a new type, you should just reference the schema for ExtendedLocation

Reply:
What is the mutability?